reyna/src/com/b2msolutions/reyna/blackout/Time.java

```diff
@@ -39,4 +39,8 @@ public boolean isBeforeOrEqualTo(Time other) {
     public boolean isAfterOrEqualTo(Time other) {
         return this.minuteOfDay >= other.minuteOfDay;
     }
+
+    public long getCurrentTimeMillis() {
+        return System.currentTimeMillis();
```

Comment:
System.currentTimeMillis() has to wrapped and not static in order to mock and test. Adding it to the Time class is purely so I didn't have to make a new class to wrap System.currentTimeMillis() as it would do nothing else. I can extract it, I just figured Time was a good enough place to stick this as it's not really specifically blackout time (apart from being in the blackout package) since it's just generic handling of time that happens to be used for our blackout stuff.